addon/templates/components/frost-select-dropdown.hbs

```diff
@@ -50,18 +50,26 @@
         {{/if}}
       </div>
     {{/if}}
-    {{#each renderItems as |item index|}}
-      {{frost-select-li
-        class=item.className
-        data=item
-        filter=filter
-        hook=(concat hook '-item')
-        hookQualifiers=(hash index=index label=item.label value=item.value)
-        id=(concat 'frost-select-list-item-' index)
-        multiselect=multiselect
-        onItemOver=(action 'focusOnItem')
-        onSelect=(action 'selectItem')
-      }}
+    {{#each renderItems key='value' as |item index|}}
+      <li
+        class={{item.className}}
+        data-test={{hook (concat hook '-item')}}
```

Comment:
Are the single select list items no longer getting hookQualifiers? That would be a major problem.